beacon.go

```diff
@@ -123,6 +122,21 @@ func genomeExists(ctx context.Context, refName string, allele string, coord int6
 	return result.Count > 0, nil
 }
 
+type where struct {
```

Comment:
If you do keep the type, you can just have 'type where string' here (you don't need a struct when there is only a single member)

Reply:
N/A